README.md

```diff
@@ -466,6 +446,36 @@ before the deployment can proceed:
 For more information and a tutorial for how to set this up, see this
 [discussion](https://github.com/cdklabs/cdk-pipelines-github/issues/162).
 
+### Comments At Top Or Bottom Of Pipeline YAML
+
+A comment can be added to the top or bottom of the pipeline. This can be used to
+add boilerplate explaining the pipeline file as well as describe what's being
+deployed.
+
+```ts
+const pipeline = new GitHubWorkflow(/* ... */);
+
+pipeline.workflowFile.commentAtTop = `AUTOGENERATED FILE, DO NOT EDIT DIRECTLY!
+See ReadMe.md for instructions and explanation.`;
+
+pipeline.workflowFile.commentAtBottom = `THE END`;
```

Comment:
I don't think its necessary to actually expose this to the user. I think we should just default to adding a comment at the top (and have no comment at the bottom). 

The only real use case for this is to say that this is an autogenerated file. I don't see any reason why users would want to modify this message in any way (or add anything to the bottom).

Reply:
I wasn't aware of any immediate use cases either, but figured it cost nothing to expose and doesn't pose a risk of invalidating the file. 🤷